From 2eedb379fa3118ca12d4ec0144e917a97adbbf71 Mon Sep 17 00:00:00 2001
From: David Clark <dclark@sierrawireless.com>
Date: Tue, 16 Jan 2018 14:38:18 -0800
Subject: [PATCH 3/4] Reduced WiFi connection timeout to 25s

The original 30s timeout is identical to the watchdog timer and thus
there was a risk that the watchdog would expire and terminate the app
while trying to connect to wifi.

Resolves: LE-9184
Change-Id: Ia7bc2a78495e12445b90ad53b9c5618bb21dea05
---
 service/platformAdaptor/ti/pa_wifi.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/service/platformAdaptor/ti/pa_wifi.sh b/service/platformAdaptor/ti/pa_wifi.sh
index d12d58c..2ae101a 100644
--- a/service/platformAdaptor/ti/pa_wifi.sh
+++ b/service/platformAdaptor/ti/pa_wifi.sh
@@ -35,7 +35,7 @@ export PATH=/legato/systems/current/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/
 # Exit with 0 if connected otherwise exit with 30 (number of retries)
 CheckConnection()
 {
-    retries=30
+    retries=25
     echo "Checking connection..."
     # Verify connection status
     for i in $(seq 1 ${retries})
@@ -253,3 +253,4 @@ case ${CMD} in
     echo "Parameter not valid"
     exit 99 ;;
 esac
+
-- 
2.11.0

